### DISCUSSION TITLE
Helmet must have pocket for flashlight, thermal goggles, glowstick, medscaner

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#11266] [Ideas and suggestions]
The helmet in the game is completely unrealistic.
It's wrong that a flashlight, a thermal goggles, a medical scanner can be attached to the head ONLY INSTEAD of a helmet.
As a result, the human head is completely unprotected from concussions.
It will be properly to attach flashlights, thermal goggles, and other things TO your helmet.
Therefore, I think a helmet should have a pocket for a flashlight, a medical scanner, thermal goggles.

**Comment by MarkCringan** [comment]
I think it is intended to work as more of a balancing act, so you have to choose between better protection & better visibility etc.

For a more lore-related reason as to why not, having all of those attachments on your helmet would make it too bulky to fit in your diving suit.
